app/components/Breadcrumbs.tsx

```diff
@@ -27,13 +26,13 @@ export const Breadcrumbs = () => {
     .filter(Boolean);
 
   const items: BreadcrumbItem[] = withCrumbs.map((match: CustomUIMatch) => {
-    const { title, icon } = match?.handle?.breadcrumb as BreadcrumbMeta;
+    const { title } = match?.handle?.breadcrumb as BreadcrumbMeta;
+    const isLast = match.id === withCrumbs[withCrumbs.length - 1].id;
```

Comment:
Nit: While reading the code I was wondering what `withCrumbs` is... wdyt update updating the name to
```suggestion
    const isLast = match.id === routesWithCrumbs[routesWithCrumbs.length - 1].id;
```
or
```suggestion
    const isLast = match.id === breadcrumbRoutes[breadcrumbRoutes.length - 1].id;
```
We can also stick with it. 🙂